test/include/concurrency/testing_transaction_util.h

```diff
@@ -175,6 +186,8 @@ struct TransactionSchedule {
   ResultType txn_result;
   std::vector<TransactionOperation> operations;
   std::vector<int> results;
```

Comment:
Name it to read_write_results to avoid confusion?

Reply:
Yeah, the name is a little bit ambiguous. We will rename that variable in future.